omega/omega-context/src/main/java/org/apache/servicecomb/pack/omega/context/annotations/SagaStart.java

```diff
@@ -37,4 +37,11 @@
    * @return
    */
   int timeout() default 0;
+
+  /**
+   * Sending out SagaEnd event to Alpha once the SagaStart annotated method is finished without any error.
+   * Default value is true, which means Omega sends out the SagaEnd event to Alpha once the annotated method is finished.
+   * Value is false, which means Omega never sends out the SagaEnd event to Alpha once the annotated method is finished.
+   */
+  boolean sendingSagaEnd() default true;
```

Comment:
I prefer to use the "delay" attribute name. It could be clear that the saga transaction will be closed later.

Reply:
I prefer `autoClose` or `async`

Reply:
+1 for autoClose.

Reply:
+1 for autoClose.

Reply:
It looks like the "autoClose" wins :)